error_test.go

```diff
@@ -218,7 +219,18 @@ func TestErrorMessages(t *testing.T) {
 			}
 
 			golden := string(tmp)
+			message = adjustMessageFormat(message)
 			assert.Equal(t, golden, message)
 		})
 	}
 }
+
+// adjustMessageFormat method modifies the message to be backward compatible with previous Go releases.
+// The change introduced in https://github.com/golang/go/commit/201cb046b745f8bb00e3d382290190c74ba7b7e1 added
+// quotas and makes some test assertions failing.
+func adjustMessageFormat(message string) string {
+	adjusted := strings.Replace(message, "unknown unit \"", "unknown unit ", 1)
+	adjusted = strings.Replace(adjusted, "\" in duration \"", " in duration ", 1)
+	adjusted = strings.Replace(adjusted, "\" accessing", " accessing", 1)
+	return adjusted
+}
```

Comment:
Which go version failed here? For go `tip` only? If we need it for tip and it is a known error in `tip` to be fixed soon I would ignore it for now. We check with `tip` to see if we might be hit by a change in reflection in the stdlib early in time.

If it fails for a released go version let's add a conditional.

Reply:
Oh, it's a bug in go1.14 :(

Checking the travis file, I noticed 1.14 is still missing.

Reply:
In this case it was "devel". I can adjust the Travis file too, unless you think this should go in a separate PR.

Reply:
Hm, if it's just tip I'd remove the function altogether. Although I'd prefer to add go 1.14 in another PR, I'd like to have some assurance that 1.14 does not contain the bug :)

Reply:
Let me prepare a PR for this.